### PR TITLE
chore: Update end-user facing documentation.

### LINKED ISSUE #69: update javascript-sdk documentation
Document all end-user facing cache client methods.

### PR DESCRIPTION
**Pull request by malandis**
Closes #69

### CODE REVIEW

File: src/SimpleCacheClient.ts [hunk 1]
```diff
@@ -16,9 +16,13 @@ export class SimpleCacheClient {
...
 
   /**
-   * @param {string} authToken
-   * @param {number} defaultTtlSeconds
-   * @param {number} requestTimeoutMs
+   * Create a SimpleCacheClient.
+   * @param {string} authToken - Momento token to authenticate requests with Simple Cache Service.
+   * @param {number} defaultTtlSeconds - A default time to live, in seconds, for cache objects
+   * created by this client.
+   * @param {number} requestTimeoutMs - An optional timeout in milliseconds for get and set operations
```
Comment (bruuuuuuuce) [inline src/SimpleCacheClient.ts:23]: This is probably my fault, but looking at this pr I notice that we are defining optional params in 3 different ways 

- saying its optional in the help text
- `{number=}`
- `{number | undefined}`

while we are updating the docs, maybe we should choose 1 approach and consolidate all the jsdocs. My vote would be for `{number=}` as its the most concise.

Comment (bruuuuuuuce) [reply src/SimpleCacheClient.ts:23]: wdyt?

Comment (malandis) [reply src/SimpleCacheClient.ts]: I opted to use the JSDoc style square bracket style  `{number} [requestTimeoutMs]`

**Review by bruuuuuuuce — APPROVED** [review]
:shipit:

**Review by tylerburdsall — APPROVED** [review]

**Pull request merged**